Jenkinsfile

```diff
@@ -198,7 +198,7 @@ node {
       stage("memcheckReleaseTest") {
         if (env.BRANCH_NAME == 'master' || full_ci) {
           sh "mkdir ./clang-release-memcheck-test"
-          sh "valgrind --tool=memcheck --error-exitcode=1 --leak-check=full --gen-suppressions=all --num-callers=25 --suppressions=resources/.valgrind-ignore.txt ./clang-release/hyriseTest clang-release-memcheck-test --gtest_filter=-NUMAMemoryResourceTest.BasicAllocate"
```

Comment:
Comment "// If this fails, run with --leak-check=full to see backtraces"?

Reply:
Sounds good to me